<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Suppliers</class>
 <widget class="QMainWindow" name="Suppliers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Suppliers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../update-server/update-server-gui/tlm_res.qrc">
    <normaloff>:/images/Icon_122.ico</normaloff>:/images/Icon_122.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="CustomQueryView" name="trvSuppliers"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QLineEdit" name="ContactName"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Address:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Email:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QTextEdit" name="Address">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>83</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="3" colspan="3">
            <widget class="QLineEdit" name="Email"/>
           </item>
           <item row="1" column="1" colspan="5">
            <widget class="QLineEdit" name="SupplierName"/>
           </item>
           <item row="0" column="1" colspan="5">
            <widget class="QLineEdit" name="SupplierNo"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Supplier No:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Company Name:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Tel:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="Telephone"/>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Mobile:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="Fax"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Fax:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QLineEdit" name="Mobile"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Contact Name:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="6">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="cmdAddNew">
               <property name="text">
                <string>Add New</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cmdSave">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cmdDelete">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="trvItemList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Requisition No</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date Requested</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Requested By</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Department</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date Required</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>No. Of Items</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File </string>
    </property>
    <addaction name="action_NEw"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Delete"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_NEw">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../update-server/update-server-gui/tlm_res.qrc"/>
 </resources>
 <connections/>
</ui>
